README.rst

```diff
@@ -64,7 +64,10 @@ psycopg2 is a dependency of Django and installing the psycopg2 wheel will likely
     The following environment variables can be set in the koku repo's .env file
         LDFLAGS="-L/usr/local/opt/openssl/lib"
         CPPFLAGS="-I/usr/local/opt/openssl/include"
-
+    Run the following commands:
```

Comment:
Alternatively if you have them in your .env file you need only `source .env` to load the environment variables.